### PR TITLE
TM-5070 Pt1 - Renaming Bureau Exceptions functions, file names, and comments

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
These PRs will just be for renaming. The heavier updates will come in forks of these PRs.

Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2762)
- ~[Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/415)~ - already merged in

[TM-5070](https://metaphase.atlassian.net/browse/TM-5070)

---
Requested Changes will be addressed in forked PR

[TM-5070]: https://metaphase.atlassian.net/browse/TM-5070?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: talentmap_api/fsbid/urls/bureau_exceptions.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+from rest_framework import routers
+
+from talentmap_api.fsbid.views import bureau_exception_list as views
+
+router = routers.SimpleRouter()
+urlpatterns = [
+    url(r'^$', views.FSBidBureauExceptionsView.as_view(), name='FSBid-bureau-exceptions'),
+    url(r'^metadata/$', views.FSBidBureauExceptionsUserMetaDataView.as_view(), name='FSBid-bureau-exceptions-user-metadata'),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/urls/bureau_exceptions.py:9]: i dont love the `metadata` but really could not think of something better

**Comment by elizabeth-jimenez** [comment]
One thing I really wanted to do with all of this renaming is consolidating Bureau Exception, Bureau Exceptions, Bureau Exception List, etc all to Bureau Exceptions

File: talentmap_api/fsbid/services/bureau_exceptions.py [hunk 2]
```diff
@@ -113,79 +115,83 @@ def add_bureau_exception_list_req_mapping(request):
...
 
-def delete_bureau_exception_list_req_mapping(request):
+def update_user_bureau_exceptions_req_mapping(request):
     return {
-        'pv_api_version_i': '',
+        'PV_API_VERSION_I': '',
         'PV_AD_ID_I': '',
         'i_pv_id': request.get('pvId'),
         'i_emp_hru_id': request.get('id'),
+        "i_PV_VALUE_TXT": request.get('bureauCodeList'),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/bureau_exceptions.py:148]: i only moved these functions around, to match their order in urls and views files, but it looks like I changed their content

File: talentmap_api/fsbid/services/bureau_exceptions.py [hunk 3]
```diff
@@ -113,79 +115,83 @@ def add_bureau_exception_list_req_mapping(request):
...
         'i_last_update_id': request.get('lastUpdatedUserID'),
         'i_last_update_date': request.get('lastUpdated'),
     }
 
-def delete_bureau_exception_list_res_mapping(data):
+def update_user_bureau_exceptions_res_mapping(data):
     if data is None or (data['O_RETURN_CODE'] and data['O_RETURN_CODE'] is not 0):
-        logger.error(f"Fsbid call for Bureau Exception List Delete failed.")
+        logger.error(f"FSBid call for Updating User Bureau Exceptions failed.")
```
Comment (M00staff) [inline talentmap_api/fsbid/services/bureau_exceptions.py:155]: i get a warning here - not sure if it matters

image

File: talentmap_api/fsbid/services/bureau_exceptions.py [hunk 4]
```diff
@@ -3,31 +3,31 @@
...
     return {
         'PV_API_VERSION_I': '',
         'PV_AD_ID_I': '',
     }
 
-def bureau_exception_list_res_mapping(data):
+def bureau_exceptions_res_mapping(data):
     if data is None or (data['O_RETURN_CODE'] and data['O_RETURN_CODE'] is not 0):
-        logger.error(f"Fsbid call for Bureau Exception List failed.")
+        logger.error(f"FSBid call for Bureau Exceptions failed.")
```
Comment (M00staff) [inline talentmap_api/fsbid/services/bureau_exceptions.py:30]: i get the same warning on all these logger statements

**Review by M00staff — APPROVED** [review]
the naming all matches up - this is pretty straight forward

👍

File: talentmap_api/fsbid/services/bureau_exceptions.py (hunk 4 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/bureau_exceptions.py:30]: oh nice find! I'll make sure to address these in the next batch of PRs

**Pull request merged**